.evergreen/config.yml

```diff
@@ -358,8 +358,7 @@ functions:
           set -o xtrace
           ${PREPARE_SHELL}
 
-          alias python=${PYTHON_BINARY}
-          python -m tox -e test-mockupdb
+          bash -i ${PROJECT_DIRECTORY}/.evergreen/tox.sh ${PYTHON_BINARY} -e test-mockupdb
```

Comment:
What does `-i` do?

Reply:
Makes the script execution interactive, so aliases defined inside of the script will correctly propagate to subprocesses created inside of it.

Reply:
Could you add a comment explaining that subtlety?

Reply:
Are you sure that the alias is needed at all? In my testing, tox sets "python" to the interpreter that tox was installed in, so there should be no need for an alias IIUC.